docs/source/examples.html.md.erb

```diff
@@ -101,6 +101,14 @@ Templates are functions called with a [suggestion object](./documentation.html#s
 <%= partial '/partials/examples/templates' %>
 ```
 
+### Instantsearch.js
```

Comment:
Well it's capitalized because that's a rule of the language :) But if you insist I will change that :)